Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/BoundsControl.cs

```diff
@@ -6,12 +6,11 @@
 using UnityEngine;
 using UnityEngine.Events;
 using UnityPhysics = UnityEngine.Physics;
-using Microsoft.MixedReality.Toolkit.Experimental.UI.BoundsControlTypes;
+using Microsoft.MixedReality.Toolkit.UI.BoundsControlTypes;
 using Microsoft.MixedReality.Toolkit.Utilities;
-using Microsoft.MixedReality.Toolkit.UI;
 using Microsoft.MixedReality.Toolkit.Experimental.Physics;
 
-namespace Microsoft.MixedReality.Toolkit.Experimental.UI.BoundsControl
+namespace Microsoft.MixedReality.Toolkit.UI.BoundsControl
```

Comment:
Nit: Reminder on the HelpURL thing here (i.e. the todo!)

It might be helpful to have a link to the issue so when we close it we can do a double check that it doesn't exist:
https://github.com/microsoft/MixedRealityToolkit-Unity/issues/7358

Reply:
added bullet point to issue - adding link to issue in cs file on next pr that fixes the handles issue